tests/lib/rules/require-computed-property-dependencies.js

```diff
@@ -727,6 +733,25 @@ ruleTester.run('require-computed-property-dependencies', rule, {
         },
       ],
     },
+    {
+      // Should ignore the left side of an assignment but not the right side.
+      code: `
+        Ember.computed(function() {
+          this.left = this.right;
+        })
+      `,
+      output: `
+        Ember.computed('right', function() {
+          this.left = this.right;
```

Comment:
Is there a separate lint rule against side effects like this in computed properties?